# tst_diskless.py fails and tst_dap.py requires network
--- a/test/run_all.py
+++ b/test/run_all.py
@@ -16,9 +16,6 @@ if python3:
 else:
     test_files.remove('tst_unicode3.py')
     sys.stdout.write('not running tst_unicode3.py ...\n')
-if __netcdf4libversion__ < '4.2.1' or __has_parallel4_support__ or __has_pnetcdf_support__:
-    test_files.remove('tst_diskless.py')
-    sys.stdout.write('not running tst_diskless.py ...\n')
 if not __has_nc_inq_path__:
     test_files.remove('tst_filepath.py')
     sys.stdout.write('not running tst_filepath.py ...\n')
@@ -29,14 +26,11 @@ if not __has_cdf5_format__ or struct.calcsize("P") < 8:
     test_files.remove('tst_cdf5.py')
     sys.stdout.write('not running tst_cdf5.py ...\n')
 
-# Don't run tests that require network connectivity
-if os.getenv('NO_NET'):
-    test_files.remove('tst_dap.py');
-    sys.stdout.write('not running tst_dap.py ...\n')
-else:
-    # run opendap test first (issue #856).
-    test_files.remove('tst_dap.py')
-    test_files.insert(0,'tst_dap.py')
+test_files.remove('tst_diskless.py')
+sys.stdout.write('not running tst_diskless.py ...\n')
+
+test_files.remove('tst_dap.py');
+sys.stdout.write('not running tst_dap.py ...\n')
 
 # Build the test suite from the tests found in the test files.
 testsuite = unittest.TestSuite()
